### PR TITLE
Chaos demo

### PR DESCRIPTION
**Pull request by thomaskwscott**

### CODE REVIEW

**Review by framiere — APPROVED** [review]
LGTM

File: chaos/Readme.md [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+    --data-raw '{
+        "config": { 
+          "topics": ["conduktor_topic_duplicate"], 
+          "duration": 1, 
+          "rateInPercent": 100, 
+          "quietPeriod": 1, 
+          "timeUnit": "MINUTES" 
+        },
+        "direction": "REQUEST",
+        "apiKeys": "PRODUCE"
```
Comment (vuduchailinh) [inline chaos/Readme.md:185]: Some interceptors will default apikey and direction. will not need to pass them in when configuring.
[InterceptorType](https://github.com/conduktor/conduktor-proxy/blob/main/proxy/src/main/java/io/conduktor/proxy/model/InterceptorType.java)

**Review by vuduchailinh — APPROVED** [review]
LGTM. Just one tiny comment

File: chaos/Readme.md (hunk 1 above)
Comment (thomaskwscott) [reply chaos/Readme.md:185]: That's cool, I like the explicit definition for the demo as it gives users a really clear idea of what is being applied.

**Pull request merged**